apps/ledger-live-desktop/src/renderer/components/Onboarding/Screens/Tutorial/index.tsx

```diff
@@ -468,7 +469,16 @@ export default function Tutorial({ useCase }: Props) {
           setConnectedDevice,
         },
         canContinue: !!connectedDevice,
-        next: () => history.push("/"),
+        next: () => {
+          if (
+            connectedDevice.modelId === DeviceModelId.nanoX ||
+            connectedDevice.modelId === DeviceModelId.nanoFTS
+          ) {
+            history.push("/onboarding/sync/completion");
+          } else {
+            history.push("/");
+          }
+        },
```

Comment:
Sorry I don't understand that, can you explain why you are doing this please?

Reply:
So from my point of view this logic of redirecting to the post onboarding only if there are actions is already handled by the `useStartPostOnboardingCallback` hook, which itself calls the `navigateToPostOnboardingHub` function that you provide in the context.
Therefore the logic should be simpler: just call the function returned by `useStartPostOnboardingCallback`, no? 🤔 

Also the navigation to `"/onboarding/sync/completion"` I don't really understand because you are in the normal onboarding here no?